.github/workflows/on-release-published.yml

```diff
@@ -120,7 +120,7 @@ jobs:
       contents: read
     steps:
       - name: Get Vault secrets
-        uses: grafana/shared-workflows/actions/get-vault-secrets@main
+        uses: grafana/shared-workflows/actions/get-vault-secrets@75804962c1ba608148988c1e2dc35fbb0ee21746
```

Comment:
Can we pls add a comment with the version that the commit hash is related to?

Reply:
These are the current latest commit in main though that will obviously no longer be the case when the next commit is made. Do you think I should pin them to the latest release version instead?

Reply:
I see now, I didn't realise the comments were used by another tool, I'm updating all to match `vX.X.X`